src/devices/Mcp23xxx/Mcp23008.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Device.I2c;
+
+namespace Iot.Device.Mcp23xxx
+{
+    public class Mcp23008 : Mcp230xx
+    {
+        public Mcp23008(I2cDevice i2cDevice, int? reset = null, int? intA = null, int? intB = null)
```

Comment:
for someone who doesn't know what Mcp23008 is I have no clue what intA and intB are. Any chance you could put a `///` comment on top of the constructor (rest APIs is fairly obvious)

Reply:
Updated wording.  Hopefully a little better now.